apps/log-collector/README.md

```
@@ -1,62 +1,67 @@
+| `DD_LOG_LEVEL` | Fluent Bit log level | `info`              |
 
-| Variable           | Description                    | Default | Example |
-| ------------------ | ------------------------------ | ------- | ------- |
-| `WRITE_TO_CONSOLE` | Enable console output for logs | `false` | `true`  |
+## Deployment
 
-## Deployment Example
+### Image Location
 
-See the complete deployment example in [example.sdl.yaml](./example.sdl.yaml).
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)